gateway/gateway.go

```diff
@@ -86,10 +87,18 @@ func init() {
 	loadStatusMap()
 }
 
+type writableElementT struct {
```

Comment:
Should we rename this to userWorkerBatchRequest? It will work well with older names like webBatchRequest etc.

Reply:
Done